Packs/ArcherRSA/ReleaseNotes/1_1_22.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### RSA Archer v2
+- Updated the Docker image to: *demisto/python3:3.9.6.24019*.
+- fixed an issue where the *api_endpoint* parameter was not added properly to the URL.
```

Comment:
missed it the last time...
```suggestion
- Fixed an issue where the *api_endpoint* parameter was not added properly to the URL.
```

Reply:
Use the display name `Advanced: API Endpoint`
customers can't see(and do not mind) the `name` key.